pex/installer.py

```diff
@@ -32,12 +32,12 @@ def function_wrapper(self, *args, **kw):
 
 
 class InstallerBase(object):
-  SETUP_BOOTSTRAP_HEADER = "import sys"
+  SETUP_BOOTSTRAP_HEADER = "import sys; from codecs import open"
```

Comment:
I just took this [SO question](http://stackoverflow.com/questions/10971033/backporting-python-3-openencoding-utf-8-to-python-2) as gospel in terms of speed improvements but admittedly didn't do any benchmarks - I'll push again with `io.open`